iXBRLViewerPlugin/__init__.py

```diff
@@ -300,12 +319,13 @@ def guiRun(cntlr, modelXbrl, attach, *args, **kwargs):
             for c in FEATURE_CONFIGS
             if cntlr.config.setdefault(f'{CONFIG_FEATURE_PREFIX}{c.key}', False)
         ]
+        pluginData(cntlr).builder = IXBRLViewerBuilder(cntlr, useStubViewer = True)
+        processModel(modelXbrl)
```

Comment:
This is missing the cntlr parameter.

When run from the GUI:
```shell
[viewer:exception] Exception processModel() missing 1 required positional argument: 'modelXbrl' 
Traceback ['  File "/Users/austinmatherne/git/Arelle/arelle/plugin/iXBRLViewerPlugin/__init__.py", line 323, in guiRun\n    processModel(modelXbrl)\n'] - wk-20230930.htm 
```

Reply:
Fixed in [36f1c09](https://github.com/Arelle/ixbrl-viewer/commit/36f1c09ac65b05db24ff89e8bec1062060d1882d)